src/chocolatey.resources/helpers/ChocolateyTabExpansion.ps1

```diff
@@ -35,56 +35,93 @@ function script:chocoCmdOperations($commands, $command, $filter, $currentArgumen
+    new       = "--template-name='' --output-directory='' --automaticpackage --version='' --maintainer='' packageversion='' maintainername='' maintainerrepo='' installertype='' url='' url64='' silentargs='' --use-built-in-template"
+    pack      = "--version='' --output-directory=''"
+    push      = "--source='' --api-key='' --timeout=''"
+    source    = "--name='' --source='' --user='' --password='' --priority= --bypass-proxy --allow-self-service"
+    config    = "--name='' --value=''"
+    feature   = "--name=''"
+    apikey    = "--source='' --api-key='' --remove"
+    export    = "--include-version-numbers --output-file-path=''"
+    template  = "--name=''"
 }
+
 $commandOptions['find'] = $commandOptions['search']
 
 try {
-    # if license exists
-    # add in pro/biz switches
+    $licenseFile = Get-Item -Path "$env:ChocolateyInstall\license\chocolatey.license.xml" -ErrorAction Ignore
+
+    if ($licenseFile) {
+        # Add pro-only commands
+        $script:chocoCommands = @(
+            $script:chocoCommands
+            'download'
+            'optimize'
+        )
+
+        $commandOptions.download = "--internalize --internalize-all-urls --ignore-dependencies --installed-packages --ignore-unfound-packages --resources-location='' --download-location='' --outputdirectory='' --source='' --version='' --prerelease --user='' --password='' --cert='' --certpassword='' --append-use-original-location --recompile --disable-package-repository-optimizations"
+        $commandOptions.sync = "--output-directory='' --id='' --package-id=''"
+        $commandOptions.optimize = "--deflate-nupkg-only --id=''"
+
+        # Add pro switches to commands that have additional switches on Pro
+        $proInstallUpgradeOptions = " --install-directory='' --package-parameters-sensitive='' --max-download-rate='' --install-arguments-sensitive='' --skip-download-cache --use-download-cache --skip-virus-check --virus-check --virus-positives-minimum='' --deflate-package-size --no-deflate-package-size --deflate-nupkg-only"
+
+        $commandOptions.install += $proInstallUpgradeOptions
+        $commandOptions.upgrade += $proInstallUpgradeOptions + " --exclude-chocolatey-packages-during-upgrade-all --include-chocolatey-packages-during-upgrade-all"
+        $commandOptions.new += " --build-package --use-original-location --keep-remote --url='' --url64='' --checksum='' --checksum64='' --checksumtype='' --pause-on-error"
+        $commandOptions.pin += " --note=''"
+
+        # Add Business-only commands and options if the license is a Business or Trial license
+        [xml]$xml = Get-Content -Path $licenseFile.FullName
+        $licenseType = $xml.license.type
+
+        if ($licenseType -in 'Business', 'BusinessTrial') {

```

Comment:
```suggestion
        if ('Business', 'BusinessTrial' -contains $licenseType) {

```
Need to use `-contains` because `-in` wasn't introduced until [PowerShell 3](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comparison_operators?view=powershell-7.3#-in-and--notin)

Reply:
Good catch, thanks lol